openapi/paths/tags-configurations.yaml

```
@@ -0,0 +1,54 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - Tags
+  summary: Retrieve tags configuration list
```

```suggestion
  summary: Retrieve tags configuration list
```
Should it be called tags configurations or tags rules?

---

Same for the path, should it be tags-rules?

---

Renamed to rule

---

I don't see the rename

---

I also meant to rename the path from `tags-configurations` to `tags-rules`.